node/as/http.js

```diff
@@ -156,13 +164,14 @@ TChannelHTTP.prototype.sendResponse = function send(buildResponse, hres, callbac
 
 TChannelHTTP.prototype.setHandler = function register(tchannel, handler) {
     var self = this;
+    self.logger = tchannel.logger;
     tchannel.handler = new AsHTTPHandler(self, tchannel, handler);
     return tchannel.handler;
 };
 
-TChannelHTTP.prototype.forwardToTChannel = function forwardToTChannel(tchannel, hreq, hres, tchannelRequestOptions, callback) {
+TChannelHTTP.prototype.forwardToTChannel = function forwardToTChannel(tchannel, hreq, hres, requestOptions, callback) {
     var self = this;
-
+    self.logger = self.logger || tchannel.logger;
```

Comment:
as-http will always use the same logger as tchannel, so logger is not passed in from TChannelHTTP constructor.